### PR TITLE
Adding RTA instructions and Kibana rules mapped to the RTA scripts

### PR DESCRIPTION
**Pull request by hungnguyen-elastic**
This PR contains:
1. The instructions to use Red Team Automation (RTA)
2. I validated so far all of the windows rules with Elastic Endpoint agent with a policy that contains the default settings of modules system, windows and endpoint security. The steps to generate
   - Step 1: Git clone the repo to a test windows machine
   - Step 2: Run each RTA script at a time
   - Step 3: Validate in Kibana UI whether rule(s) are triggered. If yes, then add signal.rule.name to the RTA python script
   - Step 4: repeat step 2 and 3

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,6 +86,33 @@ The [contribution guide](CONTRIBUTING.md) describes how to use the `create-rule`
 
 For more advanced command line interface (CLI) usage, refer to the [CLI guide](CLI.md).
 
+## Red Team Automation
```
Comment (rw-access) [inline README.md:89]: thanks for adding this!
i'm wondering if this makes sense in a separate file.
also i'm wondering if a `rta/README.md` file would make sens

File: README.md [hunk 2]
```diff
@@ -82,6 +86,33 @@ The [contribution guide](CONTRIBUTING.md) describes how to use the `create-rule`
...
+
+The repo comes with some red team automation([RTA](./rta)) python scripts that runs on Windows, Mac OS, and *nix.
```
Comment (rw-access) [inline README.md:91]:
```suggestion
The repo comes with some red team automation ([RTA](./rta)) python scripts that runs on Windows, Mac OS, and *nix. 
```

File: README.md [hunk 3]
```diff
@@ -82,6 +86,33 @@ The [contribution guide](CONTRIBUTING.md) describes how to use the `create-rule`
...
 For more advanced command line interface (CLI) usage, refer to the [CLI guide](CLI.md).
 
+## Red Team Automation
+
+The repo comes with some red team automation([RTA](./rta)) python scripts that runs on Windows, Mac OS, and *nix. 
+
+```console
+$   python -m rta -h
+
+Monday, January 18, 2021 8:57:10 PM
```
Comment (rw-access) [inline README.md:96]: we could probably trim these out
```suggestion
```

File: README.md (hunk 2 above)
Comment (rw-access) [reply README.md:91]: also i think we could add a brief sentence to explain what RTA exists and why. i think we'll find a lot of users that are unfamiliar with RTA

File: README.md [hunk 4]
```diff
@@ -10,10 +10,14 @@ This repository was first announced on Elastic's blog post, [Elastic Security op
...
-- [How to contribute](#how-to-contribute)
-- [Licensing](#licensing)
+- [Detection Rules](#detection-rules)
+  - [Table of Contents](#table-of-contents)
+  - [Overview of this repository](#overview-of-this-repository)
+  - [Getting started](#getting-started)
+  - [Red Team Automation](#red-team-automation)
+  - [How to contribute](#how-to-contribute)
+  - [Licensing](#licensing)
+  - [Questions? Problems? Suggestions?](#questions-problems-suggestions)
```
Comment (rw-access) [inline README.md:20]: good catch!

File: README.md [hunk 5]
```diff
@@ -10,10 +10,14 @@ This repository was first announced on Elastic's blog post, [Elastic Security op
 
 
 ## Table of Contents
-- [Overview of this repository](#overview-of-this-repository)
-- [Getting started](#getting-started)
-- [How to contribute](#how-to-contribute)
-- [Licensing](#licensing)
+- [Detection Rules](#detection-rules)
```
Comment (rw-access) [inline README.md:13]: this seems redundant to me, so I think we could remove this one and unindent the others

File: README.md [hunk 6]
```diff
@@ -10,10 +10,14 @@ This repository was first announced on Elastic's blog post, [Elastic Security op
...
+  - [Table of Contents](#table-of-contents)
```
Comment (rw-access) [inline README.md:14]: this too

File: README.md (hunk 1 above)
Comment (hungnguyen-elastic) [reply README.md:89]: I like that Idea, I can certainly do that

File: README.md (hunk 5 above)
Comment (hungnguyen-elastic) [reply README.md]: Honestly I don't know how these lines made it into the file. Removing them
[Table of Contents](#table-of-contents)
[Detection Rules](#detection-rules)

File: README.md (hunk 6 above)
Comment (hungnguyen-elastic) [reply README.md]: removed

File: README.md (hunk 3 above)
Comment (hungnguyen-elastic) [reply README.md]: removed

**Review by hungnguyen-elastic — COMMENTED** [review]
I made the suggested changes, let me know if you find anything else @rw-access

**Review by rw-access — APPROVED** [review]

**Comment by hungnguyen-elastic** [comment]
@brokensound77 Wondering if you had a chance to review the PR? let me know if you have any questions

File: rta/README.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+$   python -m rta -h
+usage: rta [-h] ttp_name
+
+positional arguments:
+  ttp_name
+
+optional arguments:
+  -h, --help  show this help message and exit
+```
+`ttp_name` is the name of a script that can be found in the [rta](./rta) directory. For example, to execute ./rta/wevtutil_log_clear.py script, run command:
```
Comment (brokensound77) [inline rta/README.md:19]:
```suggestion
`ttp_name` is the name of a script that can be found in the [rta](./rta) directory. For example, to execute `./rta/wevtutil_log_clear.py` script, run command:
```

**Review by brokensound77 — APPROVED** [review]
Thanks for adding this. Just one nit, otherwise LGTM 👍

**Pull request merged**